packages/nodejs/test/test.config.ts

```diff
@@ -1,11 +1,8 @@
 /**
  * @license
- * Copyright 2022-2025 Matter.js Authors
+ * Copyright 2022-2025 Project CHIP Authors
```

Comment:
should we not have anywhere matter.js authors? :-)

Reply:
Hah yeah, I must've pulled this file from my old crypto branch or something